### DISCUSSION TITLE
Open WebUI is very slow.

### DISCUSSION THREAD

**Discussion by azazar** [open-webui/open-webui#2171] [General]
It takes more than a second just to load in Brave browser from localhost. It freezes when using fast Groq LLMs.

**Comment by iukea1** [comment]
@azazar  could you provide some more details to help us troubleshoot? It would be great if you could share the output from the Google Chrome Developer Tools, especially the Network tab, so we can check the latency for each API request to openUI.

Here's a quick guide to get what we need:

1. Head over to the page in Google Chrome where the issue pops up.
2. Hit `F12` or right-click and choose "Inspect" to bring up the Developer Tools.
3. Jump to the "Network" tab.
4. Go ahead and do what you were doing when the issue happened.

Report with the latency numbers are and the waterfall requests.


Also system specs if possible en network speeds

**Comment by tandav** [comment]
Same here. I solved the issue by using postgres (via `DATABASE_URL` environment variable) instead of default SQLite. Now api request to `/chats` (which performed when you open OpenWebUI) is much faster.
